lib/internal/Magento/Framework/App/FeedOutputFormatsInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\App;
+
+interface FeedOutputFormatsInterface
```

Comment:
I don't think that it's very flexible to have  the dedicated interface for holding one constant only.